### DISCUSSION TITLE
{"code":403,"error_code":"bad_jwt","msg":"invalid claim: missing sub claim"}

### DISCUSSION THREAD

**Discussion by YordanPZ** [supabase/supabase#26568] [Questions]
Hi! when I log in with email and password, I get this error and it doesn’t allow me to access the logged-in user from the browser, nor is the accessToken set in the cookies, which does happen with the other login alternatives. On the server console, I can see the data of the logged-in user. help plz

**ERROR**
![image](https://github.com/supabase/supabase/assets/134649715/59189d03-859a-455a-ae31-ca4454261ce1)
![image](https://github.com/supabase/supabase/assets/134649715/d2573dbc-5a8a-4fa1-af96-affb0d61c1b6)

![Form](https://github.com/supabase/supabase/assets/134649715/dd25ed6e-d777-4570-878d-47d067a8d867)
![LogginButon](https://github.com/supabase/supabase/assets/134649715/c7738f80-6e3f-44da-b7d5-949dd4054100)
![LogginAction](https://github.com/supabase/supabase/assets/134649715/31b048ec-19e4-4261-af6b-44233909e2a6)
![SupabaseServer](https://github.com/supabase/supabase/assets/134649715/20fbda12-25ca-46b3-88e8-68ef89a1c8ba)

**Comment by jumploops** [comment]
I ran into this issue and tracked it down to an issue with the Supabase auth helpers and React. 

We had upgraded to `react@^18.3.1` when the issue showed up, so we pinned React at `18.2.0` and everything worked correctly.
  
Still not sure what the root cause was, but it seems that the Supabase React auth helper was looking for `useState` and couldn't find it.
